### ISSUE TITLE
LiteGuard dependency conflict

### ISSUE THREAD

**Issue by Salgat** [9swampy/Telnet#52]
## Problem
When starting a brand new .NET Core 3.1 project with the README Telnet sample, the latest version of Telnet throws the following exception,
```
System.IO.FileLoadException: 'Could not load file or assembly 'LiteGuard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null'. The located assembly's manifest definition does not match the assembly reference. (0x80131040)'
```

## Temporary Solution
Specifically add a nuget package reference to LiteGuard 2.0.1 to match expected version for .NET Core version of Telnet.

**Comment by A9G-Data-Droid** [comment]
I handled in the branch I created. My PRs went stale a long time ago and I moved on to other things. 

https://github.com/A9G-Data-Droid/Telnet/tree/UpdateLiteGuard

**Comment by Salgat** [comment]
That's disappointing to hear, I wonder why the maintainer ignored them.

**Comment by A9G-Data-Droid** [comment]
I introduced breaking changes and he doesn't want to implement changes that could effect users of this codebase. My branch is safe for new users or anyone who wants to keep working on this project.

**Comment by 9swampy** [comment]
The changes A9G suggested I did publish on the [0.9.0-beta001](https://www.nuget.org/packages/Telnet/0.9.0-beta0001). All A9G's assertions regarding the standards improvements made seem fair, but i didn't roll the changes from @toddaheath @oddmar @kaancosar over to the 0.9.0 branch. 

If it would help, create a release/0.9.1 branch from [release/0.9.0](https://github.com/9swampy/Telnet/tree/release/0.9.0), merge from master to the new 0.9.1 branch, commit any further suggestions that would help your use cases and i'll republish a 0.9.1-beta. 

I'm a little surprised that the LiteGuard dependency's still causing an issue because I did include the UpdateLiteGuard change in the [cherry pick](https://github.com/9swampy/Telnet/commit/43f10bf9af02304f3a38f6371605634bfe1d191c) from A9G's commits that rolled up to master...

**Comment by schuettecarsten** [comment]
LiteGuard has been deprecated by the author.

**Comment by A9G-Data-Droid** [comment]
Thanks @schuettecarsten . That means the way forward is to replace all the guard clauses with native methods. I don't have time to work on this right now. Hopefully someone can pick up the torch.

**Comment by 9swampy** [comment]
Included in the release 0.9.1 I've just pushed to Nuget.

Any problems, shout.
